<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeymapDialog</class>
 <widget class="QDialog" name="KeymapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnArrowDown">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblRightArrow">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>450</y>
     <width>77</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Right</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblChoice1">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>29</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choice 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblChoice2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>99</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choice 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnChoice1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblChoice3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>169</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choice 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnChoice2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblChoice4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>236</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choice 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnChoice3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblChoice5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>309</y>
     <width>60</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Choice 5</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnChoice4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblMenu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu, Mining,&lt;br&gt;  and Building</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnChoice5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblExtraInfo">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>30</y>
     <width>160</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Extra Information</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnMenu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>SPACE</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblVoice">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>100</y>
     <width>156</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Push-to-Talk Voice Chat</string>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnExtraInfo">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>R</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblArrowUp">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>240</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnVoice">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>V</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblArrowDown">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>310</y>
     <width>80</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnArrowUp">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblArrowLeft">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>380</y>
     <width>69</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Move Left</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnArrowLeft">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>LEFT</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblZoom">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>170</y>
     <width>97</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Map Overview</string>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnArrowRight">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>RIGHT</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="td::KeyButton" name="btnZoom">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>85</width>
     <height>70</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>td::KeyButton</class>
   <extends>QPushButton</extends>
   <header>../src/client/keybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeymapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeymapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
